src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The truck started working.");

```

Comment:
The message in the 'doWork' method should be more informative and indicate that it is a 'Truck' that is working. Consider including the type of machine in the message.